src/PrestaShopBundle/Controller/Admin/Sell/Address/AddressController.php

```diff
@@ -441,6 +442,13 @@ public function editOrderAction(int $orderId, string $addressType, Request $requ
             if ($result->isSubmitted() && $result->isValid()) {
                 $this->addFlash('success', $this->trans('Update successful', 'Admin.Notifications.Success'));
 
+                if ($request->query->has('submitFormAjax')) {
```

Comment:
And you need to check if `submitFormAjax` is true  ?

Reply:
I have not seen a usecase where it was passed as false 🤔 just "it's in the query"

Reply:
What's the use then `submitFormAjax` ?

Reply:
It's an old mechanism that indicates "hi, this form must be displayed inside a fancybox". This modifies the behavior of the successfull submit: instead of returning HTTP 302 to GET "view form" ; it returns a small javascript that closes the fancybox and do additional operations like passing IDs to window.parent if needed

It's quite dirty but it's how it was done before. Keeping it working allows modules to continue embedding our forms inside fancybox using this `submitFormAjax` parameter in their modules if they want to.

Reply:
Ok ... Thanks for the explanation !